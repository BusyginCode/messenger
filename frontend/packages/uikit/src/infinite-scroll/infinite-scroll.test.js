import React from 'react'
import { shallow } from 'enzyme'
import InfiniteScroll from './infinite-scroll'

describe('uikit/InfiniteScroll', () => {
  it('should renders correctly', () => {
    const onLoad = jest.fn()
    const wrapper = shallow(
      <InfiniteScroll style={{ maxHeight: 100 }} onLoad={onLoad} hasNext>
        <div style={{ height: 200 }} />
      </InfiniteScroll>
    )

    expect(wrapper).toMatchSnapshot()

    wrapper.instance().loadMore(0)

    expect(onLoad).toHaveBeenCalled()
  })
})
