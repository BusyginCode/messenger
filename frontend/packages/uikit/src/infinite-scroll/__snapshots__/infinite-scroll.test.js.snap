// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uikit/InfiniteScroll should renders correctly 1`] = `
<Fragment>
  <CustomScroll
    onScroll={[Function]}
    style={
      Object {
        "maxHeight": 100,
      }
    }
  >
    <div
      style={
        Object {
          "height": 200,
        }
      }
    />
  </CustomScroll>
</Fragment>
`;
